Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs

```diff
@@ -80,7 +84,6 @@ namespace AsterNET.Manager
 	public delegate void QueueCallerAbandonEventHandler(object sender, Event.QueueCallerAbandonEvent e);
 	public delegate void ZapShowChannelsCompleteEventHandler(object sender, Event.ZapShowChannelsCompleteEvent e);
 	public delegate void ZapShowChannelsEventHandler(object sender, Event.ZapShowChannelsEvent e);
-	public delegate void ConnectionStateEventHandler(object sender, Event.ConnectionStateEvent e);

```

Comment:
Would it not be better to have just a connectionstatechanged event and pass the connection state in the event?